README.md

```diff
@@ -164,8 +164,16 @@ statements.
 Running the Proofs
 ------------------
 
-If Isabelle is set up correctly, a full test for the proofs in this repository
-can be run with the command
+In order to open the AInvs session, the file `Kernel_Config.thy` must be generated.
+This can be done by first setting the environment variable `L4V_ARCH` to `RISCV64`
+and running
+
+    make ASpec
+
+from the directory `l4v/spec`.
+
+Once `Kernel_Config.thy` has been generated and Isabelle is set up correctly,
+a full test for the proofs in this repository can be run with the command
 
     ./run_tests
```

Comment:
Hm, this is not quite right. The full proofs can be run without doing any of that. Just a simple `./run_tests` is still fine, because it does include the `make` command, so I don't think we should change that paragraph.

Actually, reading the paragraphs below it now, it does already say that you need to use `make` for any sessions that require generated content. They just don't say which sessions these are or how to find out. So we could enumerate them. AFAIR, the sessions that directly need generated content are `ASpec`, `ExecSpec`, and `CKernel`, so if you want to build any session that depends on these, you need to use either `make` for the session you want to build (which should take care of anything it depends on) or you use `./run_tests <session>`.